lib/archethic_web/live/network_stats.ex

```diff
@@ -0,0 +1,66 @@
+defmodule ArchEthicWeb.NetworkStats do
+  @moduledoc false
+
+  use Phoenix.LiveView
+  use Phoenix.HTML
+
+  def mount(_params, _session, socket) do
+    if connected?(socket) do
+      ArchEthic.Metrics.MetricClient.monitor()
+      :timer.send_interval(5_000, self(), :update)
```

Comment:
I think it may be better to let the polling process notifying the liveview and pushing instead of pulling the points each 5s